server/lib/realtime/helpers.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Realtime.Helpers do
+  # key1=value1:key2=value2 to [{"key1", "value1"},{"key2", "value2"}]
+  @spec env_kv_to_list(String.t() | nil, keyword() | []) :: {:ok, keyword()} | :error
+  def env_kv_to_list("", _), do: :error
+
+  def env_kv_to_list(env_val, def_list) when is_binary(env_val) do
+    keywords =
+      String.split(env_val, ":")
+      |> Enum.map(
+        &(String.split(&1, "=")
+          |> List.to_tuple())
+      )
+      |> Enum.concat(def_list)
```

Comment:
@abc3 i think we should replace existing `"Content-Type"` with `"application/json"` instead of concat-ing  in case a user accidentally passes in another content type since the post body is always json in this post req: `HTTPoison.post(endpoint, serialized_txn, headers)`.

Reply:
Yes, I agree, replace is better